kore/src/Kore/Strategies/Verification.hs

```diff
@@ -94,19 +87,18 @@ import Logic
     )
 import qualified Logic
 
--- TODO (thomas.tuegel): (Pattern VariableName) should be ReachabilityRule.
-type CommonProofState = ProofState.ProofState (Pattern VariableName)
+type CommonProofState = ProofState.ProofState ReachabilityRule
 
-commonProofStateTransformer
+lhsProofStateTransformer
```

Comment:
Please add a comment describing what this `ProofStateTransformer` does. I think it gets the configuration from any proof state?